### ISSUE TITLE
BEMF term in the Current Controller model

### ISSUE THREAD

**Issue by VishnuDevA7** [geez0x1/CompliantJointToolbox#77]
The current controller model in the cjt_library_controllers.mdl contains a gain (1/k_t) that multiplies the motor velocity to compute the back-EMF compensation term. Is this right? Shouldn't the gain instead be k_t ?

Back-EMF = k_t * motor velocity

**Comment by geez0x1** [comment]
Yep that seems to be a bug... sloppy. I will try to look into this soon, but for now you can simply invert that gain.
